### ISSUE TITLE
[FEATURE] Workflow Simplification

### ISSUE THREAD

**Issue by rohansaw** [JPLMLIA/vercye_ops#6]
### Feature description

Allow users without a very technical background to easily run the pipeline, on any region in the world.

### Problem it solves

Currently, there are multiple steps of cli interaction required to run the pipeline. Also .apsimx templates need to be added manually.

### Suggested solution

We propose multiple solutions:
- Identify a way so that in the end, we ideally only have a single configuration file, in which all parameters (including path to the original shapefile) are set. And with a single script all required steps are executed.
- Setup a GUI that allows setting these parameters more interactively, also providing feedback, validation and information on the parameters.
- Include the LAI creation as part of the pipeline with deployment considerations to not recompute LAI for already existing LAI maps on the infrastructure.

**Sub-Tasks**

- [x] #7  

### Additional context

As sub-tasks, the installation should also be simplified, e.g by providing an installation script for python dependencies, apsim ...

**Comment by wronk** [comment]
Couple points:
* As we discussed in person, accomplishing this via a python notebook or some other simple GUI (that won't take much work to develop and maintain) is what I'd recommend. If we can get one of you guys (Yuval, Rohan, or Nikhil) to be able to spin up a new large-scale simulation (for a brand new region) within a few hours, I think we're in a good place. 
* I'm sure you've already seen this, but the good one-time tasks to target for inclusion into Snakemake are in the README 
* I'm open to thoughts, but my sense is that we only want to loosely couple the LAI and APSIM pipelines. The reason is that we're building to start incorporating more sources of data (S2 now, CHIRPS very soon, hyperspectral or SAR in long term), and so that will get fairly complex. A better long term solution might be to create LAI- and CHIRPS-specific data download scripts and run those on a cronjob. That way, any necessary imagery can be automatically checked/updated/downloaded every ~day w/out users having to worry about it

**Comment by Yuval-Sadeh** [comment]
I have attached the shapefiles we using for maize yield prediction in Ethiopia.
We need do start download the LAI data for theses regions.

[Ethiopia_Maize_2021_&_2019_Manually_Field_Boundaries.zip](https://github.com/user-attachments/files/18287279/Ethiopia_Maize_2021_._2019_Manually_Field_Boundaries.zip)

Mark, what do you mean by: "only want to loosely couple the LAI and APSIM pipelines"?

**Comment by rohansaw** [comment]
Agreed! I also think a notebook or a simple web-based GUI that we can deploy within the UMD Network would be good solutions. I think for the long-term especially a good description of each parameter is essential to allow new users to easily use the pipeline.

Regarding the coupling of the LAI and APSIM pipelines, I agree that loose coupling is favourable and we can leave these as two seperate pipelines. However, in terms of usability, I think it might be good to have a common entrypoint interface from which the whole LAI download and APSIM execution for a region and daterange can be started. I image it a bit as the user just having to input the shapefile with the regions, the cropmap and setting the other experiment parameters, and our application then handles the data fetching if not already available locally and subsequently starts the experiment pipeline.

Regarding the Cronjobs: Are you thinking of setting up region-specific cronjobs that just fetch and (pre-)process the newest available data on a daily (or predefined interval) base? So we define regions that we want to have a local storage of RS data and if we want to run an experiment on a new region we always add this to our download-cronjob or is it sometimes also sufficient to just once download the RS data for the daterange of the experiment and its not necessary to regularly download newer data for this region.
For the CHIRPS data a Cronjob would be nice, as the chirps files always have global coverage for a specific date and are not too large (approx 8MB), so we can just build a local store. But imo missing dates could also be fetched during execution of the pipeline if not already available locally, since the download (of new dates after an initial download of the history) is not really time-consuming.
